test/controllers/EditSetControllerSpec.scala

```diff
@@ -70,846 +69,89 @@ class EditSetControllerSpec extends BaseSpec {
-                coveringDates = "1977"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/11",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (K)",
-                coveringDates = "1975"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/10",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (J)",
-                coveringDates = "1973"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/9",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (I)",
-                coveringDates = "1971"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/8",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (H)",
-                coveringDates = "1969"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/7",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (G)",
-                coveringDates = "1967"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/2",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (A)",
-                coveringDates = "1966"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/6",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (F)",
-                coveringDates = "1965"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/3",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (C)",
-                coveringDates = "1964"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/5",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (E)",
-                coveringDates = "1963"
-              )
+        val mockMessagesApi = stubMessagesApi(messages)
+        val mockEditSet = mock[editSet]
+        val mockEditSetService = mock[EditSetService]
+        val stub = stubControllerComponents()
+        val controller = new EditSetController(
+          DefaultMessagesControllerComponents(
```

Comment:
can we use `Helpers.stubControllerComponents()` ? I think `Helpers` comes from `play.api.test`

Reply:
Done